app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -25,7 +25,8 @@ import co.nimblehq.compose.crypto.ui.uimodel.CoinItemUiModel
 @Suppress("FunctionNaming", "LongMethod")
 @Composable
 fun HomeScreen(
-    viewModel: HomeViewModel = viewModel()
+    viewModel: HomeViewModel,
```

Comment:
@hoangnguyen92dn @luongvo What do guys think If we pass `data` instead of `viewModel` here? Currently, we lost the preview in `HomeScreen.` 🤔